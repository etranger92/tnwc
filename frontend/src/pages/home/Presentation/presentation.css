@import '../../../style/share.css';
@import "../../../style/animations.css";


#presentation-container {
    height: auto;
    display: flex;
}

@media screen and (min-width: 700px) {
    #presentation-container {
        margin-bottom: 0;
    }
}

#presentation {
    margin: auto;
    width: 100%;
    display: flex;
    flex-direction: column;
    height: 100%;
    text-align: center;
    align-items: center;
    justify-content: center;
}

#presentation h5 {
    padding-top: 0px;
}


@media screen and (min-width: 700px) {
    #presentation {
        width: 60%;
    }
}

#presentation h1 {
    padding-top: 0rem !important;
}