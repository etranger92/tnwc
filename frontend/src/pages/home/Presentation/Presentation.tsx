import React from "react";
import { HashLink as LinkH } from "react-router-hash-link";
import "./presentation.css";

const Presentation: React.FC = () => {
  return (
    <section id="presentation-container">
      <article id="presentation">
        <h5>TO THE JOURNEY AHEAD</h5>
        <h1>The North West College</h1>
        <p>
          The N.W.C has been created by different teachers who aim to provide
          high quality teaching. They have been working for many years in
          different schools, colleges and universities but were never
          professionally fulfilled about the way they had to deliver or support
          their learners. When it comes to educating they have three words;
          engaging, adapting and mastering.
        </p>
        <LinkH to="/#inscription" className="link">
          Enrol
        </LinkH>
      </article>
    </section>
  );
};

export default Presentation;
