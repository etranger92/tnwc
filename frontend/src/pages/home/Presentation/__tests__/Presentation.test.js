import React from "react";
import Presentation from "../Presentation.tsx";
import { shallow } from "enzyme";
import toJson from 'enzyme-to-json';

describe("Presentation", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
 <Presentation/>

    );
  });
  test("Snapshot: Presentation", () => {
  expect(toJson(wrapper)).toMatchSnapshot();
  });
});
