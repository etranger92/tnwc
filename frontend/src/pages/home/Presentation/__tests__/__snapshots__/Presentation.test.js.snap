// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentation Snapshot: Presentation 1`] = `
<section
  id="presentation-container"
>
  <article
    id="presentation"
  >
    <h5>
      TO THE JOURNEY AHEAD
    </h5>
    <h1>
      The North West College
    </h1>
    <p>
      The N.W.C has been created by different teachers who aim to provide high quality teaching. They have been working for many years in different schools, colleges and universities but were never professionally fulfilled about the way they had to deliver or support their learners. When it comes to educating they have three words; engaging, adapting and mastering.
    </p>
    <ForwardRef
      className="link"
      to="/#inscription"
    >
      Enrol
    </ForwardRef>
  </article>
</section>
`;
